spec.html

```diff
@@ -43360,13 +43379,13 @@ <h1>Pattern Semantics</h1>
         <p>The production <emu-grammar>ExtendedAtom :: `\` [lookahead == `c`]</emu-grammar> evaluates as follows:</p>
         <emu-alg>
           1. Let _A_ be the CharSet containing the single character `\\` U+005C (REVERSE SOLIDUS).
-          1. Call CharacterSetMatcher(_A_, *false*, _direction_) and return its Matcher result.
+          1. Return CharacterSetMatcher(_A_, *false*, _direction_).
```

Comment:
The changes in B.1.4.4 Pattern Semantics ([Annex B] Pattern Semantics) should also be addressed by #2112.